cpp/modmesh/multidim/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2019, Yung-Yu Chen <yyc@solvcon.net>
```

Comment:
Nice to be 2024.

Reply:
The year remained in the past because the content was almost identical to the original file it copied from.